src/api/operations/access.fragments.graphql

```diff
@@ -85,6 +85,7 @@ fragment ProjectAccessFields on ProjectAccess {
   canManageIncomingReferrals
   canManageOutgoingReferrals
   canManageExternalFormSubmissions
+  canSplitHouseholds
```

Comment:
Is this still needed here, since we resolve it on the Enrollment fragment?

Reply:
`ManageHousehold` can receive the `project` prop either from the Enrollment fragment (when on the 'Manage Household' page for an existing enrollment) or from the Project dashboard (when on the 'Add Household' page). Scenario:
- go to Add Household page for the project, enroll someone new
- without leaving the page, try to add someone else who has a conflicting enrollment 
  - when `canSplitHouseholds` is resolved on `ProjectAccessFields`: you see the 'Conflicting Enrollment' alert and are able to join the enrollments
  - when `canSplitHouseholds` is not resolved on `ProjectAccessFields`: you see "You don't have permission to join enrollments" even if you actually do

Reply:
Ah gotcha, that makes sense. Thanks!